src/command_modules/azure-cli-billing/azure/cli/command_modules/billing/_params.py

```diff
@@ -3,9 +3,14 @@
 # Licensed under the MIT License. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------------------------
 
+
 # pylint: disable=line-too-long
-from azure.cli.core.commands import register_cli_argument
+def load_arguments(self, _):
+    with self.argument_context('billing invoice list') as c:
+        c.argument('generate_url', options_list=('--generate-download-url', '-d'), action='store_true', required=False, help='generate download url of the invoice')
```

Comment:
Please change the options_list from tuple style to list style. Applies throughout.